platform/ext/target/nordic_nrf/common/nrf_common.cmake

```diff
@@ -68,6 +68,8 @@ elseif(BUILD_UART_STDOUT)
     message(FATAL_ERROR "Configuration variable SECURE_UART1 (true|false) is undefined!")
   elseif(SECURE_UART1)
     add_definitions(-DSECURE_UART1)
+  else()
+    message(WARNING "nRF platforms likely need SECURE_UART1 enabled.")
```

Comment:
Currently, configurations without `SECURE_UART1` being set will not work on nRF platforms.
Perhaps we should then force this option to be enabled, like it was so far, and move this line:
https://github.com/NordicPlayground/trusted-firmware-m/blob/4c09829eb9307494c62e58650925af6f33f823c5/platform/ext/target/nordic_nrf/nrf5340/nrf5340.cmake#L20
before that one:
https://github.com/NordicPlayground/trusted-firmware-m/blob/4c09829eb9307494c62e58650925af6f33f823c5/platform/ext/target/nordic_nrf/nrf5340/nrf5340.cmake#L17

Actually, I'm not sure actually why it works in the current form in our current master. I guess it should be anyway corrected.
Same applies to nRF9160.

Reply:
It seems it was changed in https://github.com/NordicPlayground/trusted-firmware-m/commit/6cbeba6dc10655645d438231d3c23c674f1f4864. We can do an upstream patch to change it to

    if (NOT DEFINED SECURE_UART1)
      set (SECURE_UART1 OFF)
    endif()

Reply:
Added a commit for this.